contracts/SpokePool.sol

```diff
@@ -67,7 +70,7 @@ abstract contract SpokePool is
     RootBundle[] public rootBundles;
 
     // Origin token to destination token routings can be turned on or off, which can enable or disable deposits.
-    mapping(address => mapping(uint256 => bool)) public enabledDepositRoutes;
+    mapping(bytes32 => mapping(uint256 => bool)) public enabledDepositRoutes;
```

Comment:
Why do we need to convert this to `bytes32`? Isn't this mapping from an _origin_ chain token (which will always be a 20 byte `address`)?

Reply:
Given that all the arguments have been moved to bytes32-- a decision that you can challenge as well--this made sense from a consistency perspective. We also decided to emit all addresses in their bytes32 form, so keeping the address representation here would require extra casting. Does this make sense @mrice32 ?

Reply:
ye, there is an overall design here to make _everything_ bytes32 internally this makes reasoning about data structures and props easier and more consistent as its _always_ bytes32. we could in theory change this accordingly such that we treat all evm data as address type and type cast as soon as possible when entering methods with bytes32. very much opinonated, very much can be changed if you'll prefer the other option.